mda.go

```diff
@@ -0,0 +1,98 @@
+package nmea
+
+/**
+$WIMDA
+
+NMEA 0183 standard Meteorological Composite.
+
+Syntax
+$WIMDA,<1>,<2>,<3>,<4>,<5>,<6>,<7>,<8>,<9>,<10>,<11>,
+<12>,<13>,<14>,<15>,<16>,<17>,<18>,<19>,<20>*hh
+<CR><LF>
+
+Fields
+<1> Barometric pressure, inches of mercury, to the nearest 0.01 inch
+<2> I = inches of mercury
+<3> Barometric pressure, bars, to the nearest .001 bar
+<4> B = bars
+<5> Air temperature, degrees C, to the nearest 0.1 degree C
+<6> C = degrees C
+<7> Water temperature, degrees C (this field left blank by some WeatherStations)
+<8> C = degrees C (this field left blank by WeatherStation)
+<9> Relative humidity, percent, to the nearest 0.1 percent
+<10> Absolute humidity, percent (this field left blank by some WeatherStations)
+<11> Dew point, degrees C, to the nearest 0.1 degree C
+<12> C = degrees C
+<13> Wind direction, degrees True, to the nearest 0.1 degree
+<14> T = true
+<15> Wind direction, degrees Magnetic, to the nearest 0.1 degree
+<16> M = magnetic
+<17> Wind speed, knots, to the nearest 0.1 knot
+<18> N = knots
+<19> Wind speed, meters per second, to the nearest 0.1 m/s
+<20> M = meters per second
+*/
+
+const (
+	TypeMDA         = "MDA"
+	InchMDA         = "I"
+	BarsMDA         = "B"
+	DegreesCMDA     = "C"
+	TrueMDA         = "T"
+	MagneticMDA     = "M"
+	KnotsMDA        = "N"
+	MetersSecondMDA = "M"
+	EmptyMDA        = ""
```

Comment:
Do we need `EmptyMDA` ?

Reply:
In case of blank fields (which is valid according to the spec), and so the EnumString doesn't block it.
wanted to make it clear that way it's expected to have empty parts.